project/assets/login.less

```diff
@@ -0,0 +1,68 @@
+body {
+  width: 100%;
+  background-color: rgb(230, 230, 230);
+}
+
+header{
+  background: url("images/feedback-banner.jpg") center center no-repeat;
+  opacity: 0.8;
+  height: 600px;
+  width: 100%;
+
+  .grid {
+    height: 100%;
+  }
+}
+
+.login-section-1{
+  background-color: rgb(255, 255, 255);
+  padding: 70px;
+}
+
+.login-section-text{
+  padding: 115px;
```

Comment:
It isn't wrong to do so, by any means. I just found it strange that it's such a large value, but if you feel it looks good, I won't be one to argue. :)